### PR TITLE
CKRA6M5 Pull Req 1

### PR DESCRIPTION
**Pull request by mrvivek15**
### Thank you for contributing to the IoT Plug and Play Models repository

:memo: Please review this checklist before submission

- [x] :eyes: The models submitted in this PR follow the [DTDL Naming Guidelines](https://github.com/Azure/iot-plugandplay-models-tools/wiki/DTDL-Naming-Guidelines) for contributing to this repository.
- [x] 🕵️I have validated the models locally using the [dmr-client](https://github.com/Azure/iot-plugandplay-models-tools/tree/dev/clients/dotnet/Microsoft.IoT.ModelsRepository.CommandLine) command line tool.
- [x] ❗ I understand once the PR is approved, the models are considered immutable: any edits will require to submit new files.
- [x] ✨ When creating a new top level `dtmi:namespace`, I have completed the [PR Info Template](#pr-info-template) below.

:zap: PR validation steps are described in the tools [Wiki](https://github.com/Azure/iot-plugandplay-models-tools/wiki/Validation-Pipeline#pr-validation-checks).

## PR Info Template

When submitting models to the repository we ask that you provide as much of the following meta information around your models and related devices as possible. This info will be used to improve the _IoT Plug and Play_ ecosystem.

👇: Please replace the markdown comment examples with your own values.

### Company Info

> Info identifying your company (if applicable).

### Model Submission Goals

> Info related to broader submission goals.  

### Code Owners & Reserved Names

> Indicates GitHub alias entries to be added to the repo `CODEOWNERS` for the incoming model namespace. The codeowner is expected to be involved in subsequent DTDL model submissions against the same namespace.

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

### CODE REVIEW

**Comment by mrvivek15** [comment]
@microsoft-github-policy-service agree company="Renesas Electronics Corporation"

**Review by rido-min — CHANGES_REQUESTED** [review]
Hi @mrvivek15,

These interfaces are using the `renesas` folder, and we might like to validate you are allowed to use. 

Can you ask @TatsuyaOgawanx, who created #377 and #421 to confirm?

**Comment by mrvivek15** [comment]
Hi @ridomin

Sure , I will request @TatsuyaOgawanx regarding the same.

Thanks

**Comment by mrvivek15** [comment]
HI @TatsuyaOgawanx San,

Kindly allow me to use "renesas" folder for CKRA6M5 interface.

Thanks

File: dtmi/renesas/ra/ckra6m5/azckra6m5eth-2.json [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      "schema": "double"
+    },
+	{
+      "@type":"Telemetry",
+      "name": "magZ",
+      "schema": "double"
+    },
+	{
+      "@type":"Telemetry",
+      "name": "ICP10101-Pressure",
```
Comment (rido-min) [inline dtmi/renesas/ra/ckra6m5/azckra6m5eth-2.json:63]: the character `-` is not allowed in `name`

File: dtmi/renesas/ra/ckra6m5/azckra6m5eth-2.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+      "schema": "double"
+    },
+	{
+      "@type":"Telemetry",
+      "name": "ICP10101-Temperature",
```
Comment (rido-min) [inline dtmi/renesas/ra/ckra6m5/azckra6m5eth-2.json:68]: the character `-` is not allowed in `name`

**Review by rido-min — CHANGES_REQUESTED** [review]
fix invalid names

**Comment by TatsuyaOgawanx** [comment]
@rido-min 
Thank you for checking it.
I thought we need to review it, so we will pull request again after fixing it.

**Pull request closed without merging**